tests/test_text_generation_example.py

```diff
@@ -19,32 +21,38 @@
-            ("meta-llama/Llama-2-7b-hf", 512, False, 8711),  # in some cases like TGI, reuse_cache isnt used
-            ("stabilityai/stablelm-2-12b", 1, False, 74.8904496532218),
-            ("codellama/CodeLlama-34b-hf", 1, True, 32.644),
-            ("bigcode/starcoder2-3b", 1, False, 261.07213776344133),
-            ("adept/persimmon-8b-base", 4, False, 366.73968820698406),
-            ("Qwen/Qwen1.5-7B", 4, False, 490.8621617893209),
-            ("google/gemma-7b", 1, False, 109.70751574382221),
-            ("state-spaces/mamba-130m-hf", 1536, False, 5385.511100161605),
-            ("Deci/DeciLM-7B", 1, False, 120),
-            ("Qwen/Qwen2-7B", 512, False, 9669.45787),
-            ("Qwen/Qwen1.5-MoE-A2.7B", 1, True, 44.25834541569395),
-            ("EleutherAI/gpt-neo-2.7B", 1, False, 257.2476416844122),
+            ("bigscience/bloomz-7b1", 1, False, 130.0472971205316, False),
+            ("gpt2-xl", 1, False, 281.8734689674413, False),
+            ("EleutherAI/gpt-j-6b", 1, False, 160.5823842101192, False),
+            ("EleutherAI/gpt-neox-20b", 1, False, 50.67672679310354, False),
+            ("meta-llama/Llama-2-7b-hf", 1, True, 141.25776956002076, True),
+            ("tiiuae/falcon-40b", 1, True, 25.202450111088346, False),
+            (
+                "bigcode/starcoder",
+                256,
+                True,
+                6846.575763562658,
+                False,
+            ),  # TODO: Enable check_output after model bigcode/starcoder is fixed
+            ("Salesforce/codegen2-1B", 1, False, 446.4029486883532, False),
+            ("mosaicml/mpt-30b", 1, False, 36.06464336116623, False),
+            ("mistralai/Mistral-7B-v0.1", 1, True, 130.2172236767782, True),
+            ("mistralai/Mixtral-8x7B-v0.1", 1, False, 23.7931001677926, True),
+            ("microsoft/phi-2", 1, False, 224.72307766211117, False),
+            ("meta-llama/Meta-Llama-3-8B", 1, True, 129, False),
+            ("meta-llama/Llama-2-7b-hf", 512, True, 12808, False),
+            ("meta-llama/Llama-2-7b-hf", 512, False, 8711, False),  # in some cases like TGI, reuse_cache isnt used
+            ("stabilityai/stablelm-2-12b", 1, False, 74.8904496532218, False),
+            ("codellama/CodeLlama-34b-hf", 1, True, 32.644, False),
+            ("bigcode/starcoder2-3b", 1, False, 261.07213776344133, True),
+            ("adept/persimmon-8b-base", 4, False, 366.73968820698406, False),
+            ("Qwen/Qwen1.5-7B", 4, False, 490.8621617893209, False),
+            ("google/gemma-7b", 1, False, 109.70751574382221, True),
+            ("state-spaces/mamba-130m-hf", 1536, False, 5385.511100161605, False),
+            ("Deci/DeciLM-7B", 1, False, 120, False),
+            ("Qwen/Qwen2-7B", 512, False, 9669.45787, True),
+            ("Qwen/Qwen1.5-MoE-A2.7B", 1, True, 44.25834541569395, False),
+            ("EleutherAI/gpt-neo-2.7B", 1, False, 257.2476416844122, False),
         ],
```

Comment:
So only the model with "True" will have output tested? If we want to add more output check later on, all we need to do is change this last value to True, and add the values 1st token output to the "MODEL_OUTPUTS" table below?

Reply:
yes that is right